src/main/java/com/academy/terai/Controller/ApplicationController.java

```diff
@@ -0,0 +1,59 @@
+package com.academy.terai.Controller;
```

Comment:
Package naming - should be 'controllers'. Never start package name with capital letter.